Sabbath School/Quarterly/Controller/QuarterlyController.swift

```diff
@@ -92,16 +103,46 @@ class QuarterlyController: TableController {
         let nc = ASNavigationController(rootViewController: settings)
         self.present(nc, animated: true)
     }
+    
+    func showLessonScreen(quarterly: Quarterly) {
+        guard dataSource.contains( where: { $0 == quarterly } )  else { return }
+        
+        let lessonController = LessonWireFrame.createLessonModule(quarterlyIndex: quarterly.index) as! LessonController
+        
+        lessonController.shouldSkipToReader = true
+        self.show(lessonController, sender: nil)
+    }
 }
 
 extension QuarterlyController: QuarterlyControllerProtocol {
     func showQuarterlies(quarterlies: [Quarterly]) {
-        self.dataSource = quarterlies
-
+        guard currentQuarterlyType() == .collegiate else {
+            self.dataSource = quarterlies
+            reloadView()
+            return
+        }
+        
+        var mutableQuarterlies = quarterlies
+        
+        for quarterly in mutableQuarterlies {
+            if quarterly.id.contains("cq") {
```

Comment:
I just noticed that this might be translate well with other languages. Some languages don't have collegiate